app/assets/src/components/ui/controls/dropdowns/MultipleDropdown.jsx

```diff
@@ -25,6 +25,12 @@ class MultipleDropdown extends React.Component {
     this.passedOnChange && this.passedOnChange(e, value);
   }
 
+  componentWillReceiveProps(nextProps) {
```

Comment:
https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops

Reply:
This is not available on the current react version we are using.
Created a task for that and necessary refactoring: https://jira.czi.team/browse/IDSEQ-325